.github/workflows/deploy.yml

```
@@ -13,7 +13,7 @@ jobs:
     runs-on: ubuntu-latest
     container: "swift:5.6"
```

[IMO]
ここもupdate必要そうかな

```suggestion
    container: "swift:5.10"
```